plugins/amqp_trx_plugin/amqp_trx_plugin.cpp

```diff
@@ -123,37 +123,36 @@ struct amqp_trx_plugin_impl : std::enable_shared_from_this<amqp_trx_plugin_impl>
       const auto& tid = trx->id();
       dlog( "received packed_transaction ${id}", ("id", tid) );
 
-      auto trx_trace = fc_create_trace("Transaction");
+      auto trx_trace = fc_create_trace_with_id("Transaction", tid);
```

Comment:
This is before transaction deduplication, so if we receive the same transaction more than once, they'll get the same trace id.

Reply:
Although, maybe that's what we want?

Reply:
Yes. I think that would be desired so that they are grouped together in the zipkin ui. Note the zipkin ui does not overwrite when they have the same id, it groups them together.